mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/programmatic/PropertyMappingFieldOptionsStep.java

```diff
@@ -46,12 +49,30 @@ default S valueBridge(ValueBridge<?, ?> bridgeInstance) {
 	}
 
 	/**
+	 * Define a value binder, responsible for creating a bridge.
+	 * To pass some parameters to the bridge,
+	 * use the method {@link #valueBinder(ValueBinder, Map)} instead.
+	 *
 	 * @param binder A {@link ValueBinder} responsible for creating a bridge.
 	 * @return {@code this}, for method chaining.
 	 * @see GenericField#valueBinder()
 	 * @see ValueBinder
 	 */
-	S valueBinder(ValueBinder binder);
+	default S valueBinder(ValueBinder binder) {
+		return valueBinder( binder, Collections.emptyMap() );
+	}
+
+	/**
+	 * Define a value binder, responsible for creating a bridge.
+	 * With this method it is possible to pass a set of parameters to the binder,
+	 * so that they can be used by the bridge.
```

Comment:
```suggestion
	 * With this method it is possible to pass a set of parameters to the binder.
```


Parameters don't have to be used by the bridge, necessarily. They can just control mapping options (is this field sortable, ...).